packages/babel-generator/src/generators/modules.js

```diff
@@ -180,11 +180,15 @@ export function ImportDeclaration(node: Object) {
 
   this.print(node.source, node);
 
-  if (node.attributes?.length) {
+  if (node.assertions?.length) {
```

Comment:
nit: It seems that `import foo from "foo.json" asserts {}` are now generated as `import foo from "foo.json"`. Although it is semantically equivalent. I slightly prefer to reserve `asserts {}` if `node.assertions` is not nullish.

Reply:
I realized that current AST design does not differentiate between `import foo from "foo.json" assert {}` and `import foo from "foo.json"`. Both of them have `assertions` set to `[]`.